packages/flutter_tools/test/integration.shard/stateless_stateful_hot_reload_test.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io' as io; // ignore: dart_io_import
```

Comment:
nit: ignores need a comment explaining why this is ignored, see style guide.

Reply:
I'm going to try to fix the window issue